ansible/adhoc/cudatests.yml

```diff
@@ -7,3 +7,8 @@
     - ansible.builtin.import_role:
```

Comment:
Given we don't even run devicequery, I think we should just remove this task entirely TBH. But leave the role pending thinking more!